platform/ios/src/MGLMapView.mm

```diff
@@ -860,6 +861,13 @@ - (CGPoint)contentCenter
     return CGPointMake(CGRectGetMidX(contentFrame), CGRectGetMidY(contentFrame));
 }
 
+/// Returns edge insets with negative values replaced with 0.
+- (UIEdgeInsets)removeNegativeInsets:(UIEdgeInsets)insets
+{
+    return UIEdgeInsetsMake((insets.top > 0 ? insets.top : 0), (insets.left > 0 ? insets.left : 0),
```

Comment:
This should be expressed with the MIN macro for brevity. And since it's only used in one place, it should be inclined.